kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3Storage.java

```diff
@@ -60,14 +62,20 @@ public S3Storage(S3SinkConnectorConfig conf, String url) {
   }
 
   public AmazonS3 newS3Client(S3SinkConnectorConfig config) {
+    String proxyHost = config.getString(S3_PROXY_HOST);
+    int proxyPort = config.getInt(S3_PROXY_PORT);
+
+    ClientConfiguration clientConfiguration = PredefinedClientConfigurations.defaultConfig();
+    clientConfiguration.withUserAgentPrefix(String.format(VERSION_FORMAT, Version.getVersion()));
+    if(proxyHost != null && !proxyHost.isEmpty()) {
```

Comment:
Also, here regarding the previous comment from @ewencp : 
the previous use of withers had the advantage that, if the defaults (`null` and `-1`) were set with the "withers", aws sdk would also check for the system properties `https.proxyHost` and `https.proxyPort`. @ewencp is this you think an option we want to have along with our configuration properties? Or we should override it with our configuration regardless?